src/gui/src/CapriceAbout.cpp

```diff
@@ -17,7 +17,7 @@ CapriceAbout::CapriceAbout(const CRect& WindowRect, CWindow* pParent, CFontEngin
 	m_pMessageLabel = new CLabel(CPoint(5, 70), this, "Version 4.3.0");
 	m_pMessageLabel = new CLabel(CPoint(5, 90), this, "F1 - Menu / Pause");
 	m_pMessageLabel = new CLabel(CPoint(5, 100), this, "F2 - Fullscreen");
-	m_pMessageLabel = new CLabel(CPoint(5, 110), this, "F3 - ");
+	m_pMessageLabel = new CLabel(CPoint(5, 110), this, "F3 - Save screenshot");
 	//m_pMessageLabel = new CLabel(CPoint(5, 120), this, "F3 - Load/Save disk/tape/snapshot/screenshot");
```

Comment:
You can remove this comment (not sure why I kept it, the coordinates are not even right ...)